frontend/lib/js/category-trees/CategoryTreeFolder.js

```diff
@@ -21,6 +25,7 @@ type PropsType = {
   active: boolean,
   open?: boolean,
   onToggleOpen?: Function,
+  search: SearchType,
```

Comment:
Because the `search` is an integral part of the app. In the beginning it was optional. Correctly, it should be optional. I change it.

i.e. `open` is not always optional.